deltachat-ios/Controller/NewProfileViewController.swift

```diff
@@ -67,13 +67,6 @@ class NewProfileViewController: UIViewController, QrCodeReaderDelegate {
         return DcContact(id: Int(DC_CONTACT_ID_SELF))
     }
 
-    var fingerprint: String? {
-        if !DcConfig.configured {
-            return nil
-        }
-        return dcContext.getSecurejoinQr(chatId: 0)
-    }
-
```

Comment:
i removed this indirection as the variable "fingerprint" is actcually no fingerprint but the content of a qrcode to render - which contains things needed for a securejoin - this may or may not include a fingerprint.